### PR TITLE
ISSUE_TEMPLATE: add templates

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [ ] The page (if new), does not already exist in the repository.
- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [ ] The page has 8 or fewer examples.
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

# ToDo

- [x] bug report template
- [x] page request template
- [x] page translation request template
- [x] gitter chat link in the list of issue templates

### CODE REVIEW

**Review by pixelcmtd — COMMENTED** [review]
What exactly do we need bug reports for? Have we ever had any bugs in this repo?

**Comment by EmilySeville7cfg** [comment]
> What exactly do we need bug reports for? Have we ever had any bugs in this repo?

We may use them as requests to fix some tldr pages. It's preferred to use PRs but if a user don't know how he/she/other can use issue templates. These are my thoughts.

**Comment by CleanMachine1** [comment]
I _think_ I've reported a "bug" before, a single typo I couldn't be bothered to make a PR for

File: .github/ISSUE_TEMPLATE/page-translation.yml
Comment (SethFalco) [inline .github/ISSUE_TEMPLATE/page-translation.yml:35]:
```suggestion
      placeholder: e.g. OS version
```

Could be worth telling them how to check? 🤔 
For example, on Linux `uname -a` would be great.

I'm betting Windows/macOS have a similar thing.

Might also be out-of-scope for this template though, welcome to ignore this.

**Review by SethFalco — COMMENTED** [review]
Just a few comments.

Some of them can be doubled it, but I didn't want to spam you.
For example, where I said the `placeholder` was wordy, applies to all `placeholder` fields.

File: .github/ISSUE_TEMPLATE/page-request.yml
Comment (CleanMachine1) [reply .github/ISSUE_TEMPLATE/page-request.yml:10]: TLDR or tldr is most acceptable :+1:

File: .github/ISSUE_TEMPLATE/page-request.yml
Comment (CleanMachine1) [inline .github/ISSUE_TEMPLATE/page-request.yml:21]:
```suggestion
      description: Specify the operating system(s) this command is implemented for
```

If a user is requesting a command, they surely know what OS it works on.

File: .github/ISSUE_TEMPLATE/page-request.yml
Comment (CleanMachine1) [inline .github/ISSUE_TEMPLATE/page-request.yml:9]:
```suggestion
      description: Describe a command you want to be summarized.
```

Please apply this change throughout.

File: .github/ISSUE_TEMPLATE/page-translation.yml
Comment (EmilySeville7cfg) [reply .github/ISSUE_TEMPLATE/page-translation.yml]: Maybe it's better to use `lsb_release -a`.

Comment (EmilySeville7cfg) [reply .github/ISSUE_TEMPLATE/page-translation.yml]: Yes, it's a nice idea. 👍🏻

**Comment by EmilySeville7cfg** [comment]
@SethFalco, @CleanMachine1, you can review it again. ;)

File: .github/ISSUE_TEMPLATE/page-translation.yml
Comment (SethFalco) [inline .github/ISSUE_TEMPLATE/page-translation.yml:35]: Only part I'm partial too. I do like that we explain it for Linux users, but this is written like it would work for everyone.

I don't think that would work for Windows/macOS/Android/etc users.

I'd suggest either:
1) Clarify that only works on Linux
2) Remove it

**Review by SethFalco — COMMENTED** [review]
Everything else is nice imo. 🎉

File: .github/ISSUE_TEMPLATE/page-translation.yml
Comment (EmilySeville7cfg) [reply .github/ISSUE_TEMPLATE/page-translation.yml]: I've specified such commands for Linux, macOS, Windows.

**Review by SethFalco — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/page-translation.yml
Comment (CleanMachine1) [inline .github/ISSUE_TEMPLATE/page-translation.yml:1]:
```suggestion
name: 📄 Page translation request
```

**Review by CleanMachine1 — APPROVED** [review]
Lets get another review before merging due to the nature of this PR

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**
